### ISSUE TITLE
Project 5: Utilizing machine learning for attributing foodborne pathogens

### ISSUE THREAD

**Issue by kipkurui** [ENEZA-DSI/mini-projects-2024#5]

**Comment by adolfmukama** [comment]
# This is my first choice project.
My name is Adolf Oyesigye Mukama, **an MSc Bioinformatics student at Pwani University.** My primary interest lies in genomic epidemiology and I'm focusing on the genomic epidemiology of *Serratia marcescens* organism for my MSc project. This mini project will enhance my skills in analyzing WGS  data, essential for my Msc. project research. Additionally, over the past two weeks, we've been introduced to machine learning. **I am keen to integrate machine learning skills gained into the genomics workflow,** further enriching my Msc. project and broadening my expertise in bioinformatics.

**Comment by Kimani-kj** [comment]
# MY FIRST CHOICE  
#### MY ONLY CHOICE 
Am kariuki kimani from pwani University.

I am deeply committed to this project as it represents both my top choice and my backup plan. My primary motivation is to acquire in-depth knowledge of the genomics pipeline, specifically focusing on quality control (QC) and variant calling. This expertise is essential for my own research, which involves variant calling and differential expression analysis of asymptomatic versus symptomatic malaria variant surface antigens (VSAs).

I am also enthusiastic about integrating machine learning techniques into my work. The skills gained from the Eneza training will be invaluable in this context. This project not only aligns perfectly with my current research but also offers a unique opportunity to explore how to extend my work into predictive modeling for future VSA-related infections.

If this project is not available, project three would be a suitable alternative. However, this project remains my first choice due to its alignment with my research goals.

**Comment by Asiimwe23** [comment]
# THE PROJECT OF FIRST CHOICE
I am choosing this project because it has an interplay of both genomics and machine Learning. This will give me insights of how you can link the genomic analysis and machine learning for more pronounced insights from the data. The project am handling now requires me to develop a tool that will be used for analysis of amplicon sequencing data mainly for drug resistance surveillance. Machine learning aspect will give me more broader options of even coming up with a tool that can be easily used for genomic surveillance especially for identifying and predicting host sources of resistant genes in East Africa.  Further to this am interested on the hands on experience to apply some knowledge I have learnt about variant calling in my future learn training. I am interested in really performing feature engineering, applying and training different machine models and optimization as well. E.coli is has been a challenge in regard to food borne diseases for a long time, this work will be of great insight in the East African setting and am ready for the full experience.
Asiimwe Emmanuel
Pwani University (MSc Bioinformatics)

**Comment by Dzoro5** [comment]
I am interested in this project (first choice) because it has the aspect of genomics, which is a new learning area to me that I would like to explore in addition to getting to understand the machine learning techniques. 
Edwin Dzoro Mwazuma
Research Assistant
Aga Khan University.

**Comment by Dorothy1800** [comment]
My name is Dorothy Chepkoech, Masters fellow in Data Science at Aga Khan University Institute of Human Development.  I chose this project as my first choice because currently in my research I will be looking at predictive and prescriptive modelling of Neurodevelopmental disorders based on genetics and other factors. Doing this project will help me to learn and enhance my skills in genome sequencing. In terms of skills I am conversant with machine learning and looking forward to learn about genome sequencing in this project.

**Comment by lmwagandi** [comment]
This min project is my second choice project.
IM Lenny Mwagandi from Pwani university as a faculty member (Lecturer) in the Department of Biochemistry and Biotechnology. My background is in Biochemistry and Molecular Biology with good background in analyzing data using normal statistical packages such as mintab, Genstart, SPSS, etc.. However, I'm not very competent in Bioinformatics in the area of genomics specifically population genomics. I have interest in population genomic in foodborne pathogens. Through this project Im expecting to acquire knowledge and understand the use of machine learning tools in genomic analysis of various foodborne pathogens in trying to develop various in pipelines in bioinformatics which will help me to carry out research in identification of various foodborne pathogens along side their sources

**Comment by johanmed** [comment]
**Background information**
- Name: Johannes Medagbe
- Institution of origin: Pwani University
- Current education: MSc Bioinformatics
- Former educations: MSc Genetics, BSc Biomedical Sciences

**Research interests**
- Artificial Intelligence and Machine learning to accelerate advancements in Health Sciences
- Software engineering
- Bioinformatics

**Level of preference**

This is my **first** choice. I am highly interested in this project.

**Motivation**

My interest for this project primarily comes from the fact that it is at the intersection of Omics Data Science (Genomics) and Computing (machine-learning). My MSc project stands the same way. I am pretty sure it will be of great use for me to foster my skills in both regards. It will also offer a new view to tackle my project. Finally, as far as global health is also concerned I believe this project will lay the foundations for me to make some impact and contribute to human health.

**Skills of interest to the project**
- Experience in machine-learning
In plus of the introduction to AI and machine-learning we have all been through, I have some hands-on experience in machine-learning. I have done some **computer vision** and **regression** tasks using **Tensorflow**. I am also continuously getting better as my MSc Research project aims at using machine-learning to dissect interactions in multidimensional data of GeneNetwork resource.
- Bioinformatics skills
Through my Bioinformatics training, I have some **good knowledge and command of Omics Analysis**.
- Biomedical experience
I have some good experience in Biomedical Sciences as it is my original area of study. My undergraduate project was actually at the intersection of Microbiology and Immunology.

**Comment by Parcelli** [comment]
**FIRST CHOICE**

I am Parcelli Jepchirchir, currently pursuing MSc Bioinformatics at Pwani University.I am interested in application of genomics techniques in improving human health.

**Reasons for choosing the project**

I am interested in this project because its a blend of both genomics and machine learning. I have experience working with genomics data and I look forward to applying machine learning skills from the training in this project.
The project presents an opportunity to gain hands on skill in use of  machine learning specifically to genomics data.The project is  an opportunity for growth .

**Comment by Rodneyomukuti** [comment]
Name: Rodney Omukuti
Institution: Pwani University
Background: Biological Sciences & Bioinformatics

Project preference: **First**

**Reason(s):**

**1. Aligns with my career goal:**  It combines my interests in genomics and machine learning, which will help me to gather skills and develop expertise as a budding data analyst.

**2. Project focus:** The project focuses on utilizing underexplored WGS datasets from low-resource countries presenting an opportunity to contribute to an underserved area of research.

**3. Potential impact of the project:** Successful development of predictive models for E. coli source attribution can have a meaningful impact on public health interventions, particularly in low-resource settings.

**Comment by Puritynjenga** [comment]
#### FIRST CHOICE
My name is Purity Njenga an Intern at ICIPE. I have a background in Biotechnology. My curiosity for studying disease causing pathogens makes this project my first choice. I would like to leverage the available resources and data to be able to work on the project and contribute to food safety.

Utilizing machine learning to trace sources of foodborne pathogens, excites me for several reasons. By leveraging advanced computational techniques, I can enhance the accuracy of identifying pathogen sources, contributing significantly to public health and food safety. This innovative approach with cutting-edge technology, offers a chance to solve real-world problems.

**Comment by karegi** [comment]
Name:  Mputhia Milka
Education Background: BSc Genomics & MSc Bioinformatics 
Institution Affiliation: Pwani University 
1st Choice 

Why I am choosing this project:

I am motivated to pursue this project as it addresses crucial aspects of foodborne disease surveillance in East Africa through the application of machine learning to E. coli source identification. Leveraging my skills in genomics and bioinformatics, to analyze public WGS datasets to pinpoint genetic markers associated with E. coli host sources. The insights gained will significantly bolster my MSc research, equipping me with expertise in genomic data analysis and predictive modeling essential for advancing epidemiological studies in public health.

**Comment by Ojwang-Biot** [comment]
**1st Choice**
I am Paul Ajwang from Pwani University. I have a background in Biotechnology and Bioinformatics. I am expressing my interest towards this project due to the hybrid nature of the project incorporating aspects of machine learning and bioinformatics. Currently, MSc. project involves constructing a pangenome, which serves as a foundation for future genetic studies. It will be interesting to use the the skills learnt by participating in this project to perform the downstream analysis especially identifying genetic variations in pangenome using machine learning, which is not popular currently. 

Collaborating in the project will be a plus for me in utilizing machine learning and genomics to current and future challenges.

**Comment by DOREENKDAVID** [comment]
**FIRST CHOICE**
I am Doreen Kinya, MSc Bioinformatics student at Pwani University. Taking on this project would allow me to deepen my understanding of bioinformatics and machine learning, particularly in applying these techniques to my project on comparative genomic analysis of Nipah virus and SARS-CoV-2. Through the thorough process of data acquisition, quality control, variant calling, and feature engineering, I will improve my technical abilities and proficiency in genomic analysis by utilizing Python modules such as Biopython, TensorFlow, Keras, scikit-learn, and Matplotlib.

**Comment by OdongoIsaya** [comment]
**MY FIRST CHOICE-ISAYA ODONGO (PWANI UNIVERSITY)**
I am Isaya Odongo an MSc student in Bioinformatics, my current project delves into exploring the intricate dynamics of pathogen colonization on host microbiome composition, diversity, abundance, and co-occurrence networks. My current approach  relies on intensive metagenomics pipelines and advanced statistical techniques. While these approaches are robust, they are inherently time-consuming and demand specialized expertise. By integrating whole genome sequencing (WGS) data with machine learning techniques, I aim to enhance my project’s capabilities in understanding host-microbiome-pathogen dynamics swiftly and accurately. This s approach holds the promise of rapidly characterising how microbial communities within hosts influence pathogenicity and overall host health and ultimately food security incase of crop plant systems.

This project represents an exciting opportunity to apply and expand upon my machine learning knowledge, contributing to advancements in understanding host-microbiome interactions across both human and plant systems .

**Comment by yiasei** [comment]
**Third Option**
My name is Yiakon Sein, a MSc Bioinformatics student at Pwani University. This mini project will offer me an opportunity to learn how to analyse Whole Genome Sequencing (WGS) data and make use of Machine learning techniques to model trends in metagenomics.

**Comment by JoelALukwe** [comment]
My names are Joel Alukwe, Institution Africa International University my background is in information technology with a specialization in Software engineering, Main focus being backend engineering.

I'm excited about Project 5 as my 1st choice for its potential in using technology to attribute foodborne pathogens to their sources, amidst complex factors like environmental conditions and food handling practices.

I hope to deepen my understanding of epidemiology effectively applying machine learning models in public health, Developing robust models that would accurately identify sources of foodborne pathogens improving enhancement of strategies in managing and preventing outbreaks, thus improving public health outcomes.

Concepts I look forward to master are
* Machine Learning( Advanced model development, evaluation, and optimization for epidemiological applications.)
* Artificial Intelligence (Practical implementation of AI algorithms to bolster food safety measures.)
* Data Science( Proficiency in sophisticated data preprocessing techniques specific to epidemiological datasets.)

**Comment by kipkurui** [comment]
@Ojwang-Biot , the whole genome assembly might be a better fit for your needs

**Comment by Ojwang-Biot** [comment]
@kipkurui thanks for the advise. I'll join the whole genome assembly group.
I was also looking forward to learn the ML part, which I believe, I will during the presentation

**Comment by kipkurui** [comment]
@Rodneyomukuti , please comment on project 2 for assignment.
